build/Linux/pencil

```diff
@@ -1,18 +1,31 @@
 #!/bin/sh
 
-
+# Target for application ini
 APP_INI="/usr/share/evolus-pencil/application.ini"
 
-if [ -x /usr/bin/xulrunner ]; then
+# List of firefox binaries to search
+BINARIES="xulrunner iceweasel firefox firefox-bin"
 
-    /usr/bin/xulrunner "$APP_INI" "$@"
+# Assembling run command
+RUN=""
 
-elif [ -x /usr/bin/iceweasel ]; then
+for BIN in $BINARIES; do
+	BIN_PATH=$(which ${BIN} 2>/dev/null)
 
-    /usr/bin/iceweasel --no-remote --app "$APP_INI" "$@"
+	if [ "$?" == "0" ]; then
+		OPTS="${APP_INI}" "$@"
 
-else
+		if [ "${BIN}" != "xulrunner" ]; then
+			OPTS="--app ${OPTS}"
```

Comment:
Turns out I had enabled it in an attempt to fix PNG exports(was a suggested fix at #170). The only linux distros I've tested pencil on are arch & nixos, both use xulrunner. Tried it with firefox & indeed it was broken. So thanks - no action needed on your part.